toonz/sources/toonz/preferencespopup.cpp

```diff
@@ -278,6 +278,35 @@ void PreferencesPopup::onPixelsOnlyChanged(int index) {
 
 //-----------------------------------------------------------------------------
 
+void PreferencesPopup::onProjectRootChanged() {
+  int documents = m_projectRootDocuments->isChecked() ? 1 : 0;
+  int desktop   = m_projectRootDesktop->isChecked() ? 1 : 0;
+  int stuff     = m_projectRootStuff->isChecked() ? 1 : 0;
+  int custom    = m_projectRootCustom->isChecked() ? 1 : 0;
+
+  QString projects = QString::number(documents) + QString::number(desktop) +
+                     QString::number(stuff) + QString::number(custom);
+  m_pref->setProjectRoot(projects.toInt());
+  if (custom) {
```

Comment:
You are using decimal (base-10) flag. I think it would be faster to use bit flags instead.